cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -84,7 +84,8 @@ describe("tests on annotation page ui with programmatic state loading and store
 
                     // B. Select the 'float' transformation heuristic
                     // :data-cy="'selectTransform_' + columnName"
-                    cy.get("[data-cy='selectTransform_age']").click().type("float{enter}");
+                    cy.get("[data-cy='selectTransform_age']").click();
```

Comment:
I think in case of typing action with `v-select`, there is no need to preface it with a `click` action.

Reply:
Looks green, @rmanaem. Committing the change and will merge.